content/docs/cookbooks/index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Cookbooks
```

Comment:
I believe we were going with guides

Reply:
I presented the cookbook concept, which is a well established genre of documentation, to Julius and RichiH and did not meet any objections. If you'd like us to reconsider, please let us know.

Reply:
When this was discussed a few months back the name we were going to go with was "guides".  As guides are a superset of cookbooks, I'd rather we not end up with both.